nntrainer/compiler/tflite_interpreter.cpp

```diff
@@ -35,6 +35,11 @@
 
 static constexpr const char *FUNC_TAG = "[TFLITE INTERPRETER] ";
 
+// This Variables need for create tflite nodes
+nntrainer::TfOpNode::Variables new_variable;
+nntrainer::Tensor new_weight_add[50];
```

Comment:
Is there any reason to use hard-coded value, 50?

Reply:
There is no particular reason. I declared static because there was no logic to calculate how many nodes to be added.
i will fix this part, Thank you for Review